calabash-cucumber/lib/calabash-cucumber/keyboard_helpers.rb

```diff
@@ -188,12 +188,61 @@ def text_from_first_responder
         text
       end
 
+      # @!visibility private
+      # Returns the keyboard type as a symbol from the specified query
+      #
+      # UIKeyboardTypeDefault => :default
+      # UIKeyboardTypeASCIICapable => :ascii_capable
+      # UIKeyboardTypeNumbersAndPunctuation => :numbers_and_punctuation
+      # UIKeyboardTypeURL => :url
+      # UIKeyboardTypeNumberPad => :number_pad
+      # UIKeyboardTypePhonePad => :phone_pad
+      # UIKeyboardTypeNamePhonePad => :name_phone_pad
+      # UIKeyboardTypeEmailAddress => :email
+      # UIKeyboardTypeDecimalPad => :decimal
+      # UIKeyboardTypeTwitter => :twitter
+      # UIKeyboardTypeWebSearch => :web_search
+      #
+      # @raise [RuntimeError] if there is no visible keyboard
+      def keyboard_type(query)
```

Comment:
We basically just need a query to pass into for `:keyboardType` - an alternative would be to use the first responder instead. Is that more desired behavior?